### DISCUSSION TITLE
Trame and apps - both mobile and desktop?

### DISCUSSION THREAD

**Discussion by nyue** [Kitware/trame#620] [Q&A] [answered]
I have been reading up about Trame and attended the recent webinar.

I saw a mention of Tauri

I am looking for example of Trame being use to build desktop app for the major OS like Windows, MacOS and Linux

I am also looking for example of Trame being use to build mobile app for the iOS and Andriod

I am in the process of evaluating Trame as part of the technology stack for building an educational software platform where visualisation is one of the way we communicate new concepts.

Thank you in advanced.

**Comment by jourdain** [comment, marked as answer]
Thanks for reaching out and I hope you are having fun with trame. 

For the tauri/mobile option, I don't think the technology is there yet (I could be wrong). So while the update from 1 to 2 would be needed to get the mobile part. The main issue will be handling the __Python side-car__ for iOS / Android. 

The bundle I created for the webinar is available [here](https://github.com/Kitware/trame-tauri/tree/master/examples/07_cool_app) for reference.
